src/physics/em/BetheHeitlerInteractor.i.hh

```diff
@@ -0,0 +1,233 @@
+    Interaction result;
+    result.action      = Action::absorbed;
+    result.energy      = units::MevEnergy{0};
+    result.direction   = {0, 0, 0};
+    result.secondaries = {electron_pair, 2};
+
+    // Argument to density function for polar angle
+    real_type log_r2r3
+        = std::log(generate_canonical(rng) * generate_canonical(rng));
+    real_type density_arg = log_r2r3
+                            * BernoulliDistribution(0.25)(rng) ? 1.6 : (1 / 1.875);
+        
+    // Polar angle from the density function
+    real_type theta = 0.0977
+                      * (density_arg * std::exp(-log_r2r3)
+                         + 27 * density_arg * std::exp(-log_r2r3));
+
+    // Outgoing secondaries are electron and positron
+    electron_pair[0].def_id = shared_.electron_id;
+    electron_pair[1].def_id = shared_.positron_id;
+    // Energies of secondaries shared equally
+    electron_pair[0].energy
+        = units::MevEnergy{0.5 * epsilon * inc_energy_.value()};
+    electron_pair[1].energy = electron_pair[0].energy;
+
+    // Azimuthal angle sampled isotropically
+    UniformRealDistribution<real_type> sample_phi(0, 2 * constants::pi);
+    electron_pair[0].direction = rotate(
+        from_spherical(std::cos(theta), sample_phi(rng)), inc_direction_);
+    // Opposite direction in costheta
+    electron_pair[1].direction = rotate(
+        from_spherical(-std::cos(theta), sample_phi(rng)), inc_direction_);
+    // Calculate produced pair directions via conservation of momentum
+    for (unsigned int p = 0; p < 2; p++)
+    {
+        // Loop over directions
+        for (int i = 0; i < 3; ++i)
+        {
+            electron_pair[p].direction[i]
+                = inc_direction_[i] * inc_energy_.value()
+                  - electron_pair[p].direction[i]
+                        * electron_pair[p].energy.value();
+        }
+        normalize_direction(&electron_pair[p].direction);
+    }
+
+    return result;
+}
+
+CELER_FUNCTION real_type BetheHeitlerInteractor::impact_parameter(size_type Z,
+                                                       real_type eps) const
+{
+    return 136 * std::cbrt(1 / Z) * epsilon0_ * epsilon0_
+           / (eps * (1 - eps));
+}
+
+CELER_FUNCTION real_type BetheHeitlerInteractor::screening_phi1(real_type delta) const
+{
+    if (delta <= 1)
+    {
+        return 20.867 - 3.242 * delta + 0.625 * delta * delta;
+    }
+    return screening_phi12(delta);
+}
+
+CELER_FUNCTION real_type BetheHeitlerInteractor::screening_phi2(real_type delta) const
+{
+    if (delta <= 1)
+    {
+        return 20.209 - 1.930 * delta - 0.086 * delta * delta;
+    }
+    return screening_phi12(delta);
+}
+
+CELER_FUNCTION real_type BetheHeitlerInteractor::screening_phi12(real_type delta) const
+{
+    return 21.12 - 4.184 * std::log(delta + 0.952);
+}
+
+CELER_FUNCTION real_type BetheHeitlerInteractor::CoulombCorr(size_type Z) const
+{
+    real_type term1 = 8.0 / 3.0 * std::log(Z);
```

Comment:
`element_.log_z()`

Also the function name formatting isn't consistent with the style guide -- `coulomb_corr`?

Reply:
Removed function in favor of `ElementView::coulomb_correction`.